components/experimental/src/dimension/provider/percent.rs

```diff
@@ -10,56 +10,50 @@
 //! Read more about data providers: [`icu_provider`]
 
 use alloc::borrow::Cow;
+use icu_pattern::{DoublePlaceholderPattern, SinglePlaceholderPattern};
 use icu_provider::prelude::*;
 
+#[cfg(feature = "compiled_data")]
+/// Baked data
+///
+/// <div class="stab unstable">
+/// 🚧 This code is considered unstable; it may change at any time, in breaking or non-breaking ways,
+/// including in SemVer minor releases. In particular, the `DataProvider` implementations are only
+/// guaranteed to match with this version's `*_unstable` providers. Use with caution.
+/// </div>
+pub use crate::provider::Baked;
+
 #[icu_provider::data_struct(PercentEssentialsV1Marker = "percent/essentials@1")]
-#[derive(Default, Clone, PartialEq, Debug)]
+#[derive(Clone, PartialEq, Debug)]
 #[cfg_attr(
     feature = "datagen",
     derive(serde::Serialize, databake::Bake),
     databake(path = icu_experimental::dimension::provider::percent),
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+/// A struct including the essentials to create a Percent.
+///
+/// If an `approximate` or `explicit plus` are required, use the negative pattern as explained below:
+/// <https://www.unicode.org/reports/tr35/tr35-numbers.html#approximate-number-formatting>
+/// <https://www.unicode.org/reports/tr35/tr35-numbers.html#explicit-plus-signs>
 pub struct PercentEssentialsV1<'data> {
-    /// The index of the number placeholder in the standard pattern.
-    pub number_index: u8,
-
-    /// Prefix and suffix to apply to a percent sign when needed.
     #[cfg_attr(feature = "serde", serde(borrow))]
-    pub percent_sign_affixes: PercentAffixesV1<'data>,
+    /// Represents the standard pattern for negative percents.
+    /// NOTE: place holder 0 is the place of the percent value.
+    ///       place holder 1 is the place of the plus, minus, or approximate signs.
+    pub negative_pattern: DoublePlaceholderPattern<Cow<'data, str>>,
```

Comment:
nit: maybe this should be `signed_pattern` and the other one `unsigned_pattern`, because this is also used for positive approximate and explicit plus numbers.